fly/commands/builds.go

```diff
@@ -192,6 +192,7 @@ func (command *BuildsCommand) displayBuilds(builds []atc.Build) error {
 			{Contents: "end", Color: color.New(color.Bold)},
 			{Contents: "duration", Color: color.New(color.Bold)},
 			{Contents: "team", Color: color.New(color.Bold)},
+			{Contents: "created-by", Color: color.New(color.Bold)},
```

Comment:
We use spaces in column titles elsewhere:

```suggestion
			{Contents: "created by", Color: color.New(color.Bold)},
```

Reply:
Actually I tried "created by", but "created-by" looked better to me. Anyway, I'm ok.